package.json

```diff
@@ -110,5 +110,9 @@
     "paths": [
       "lib/ui"
     ]
+  },
+  "volta": {
+    "node": "8.16.2",
```

Comment:
Should probably use 12? AFAIK, there is no consumer facing impact to changing, right?